### ISSUE TITLE
Upgrade from 2.1.27.0 to 2.1.27.4 cannot procceed with an empty SALT key

### ISSUE THREAD

**Issue by mrsaraiva** [nilsteampassnet/TeamPass#1801]
### Steps to reproduce
1. Extract 2.1.27.4 to a new directory
2. Rename old directory with 2.1.27.0 
3. Copy /includes/config/settings.php,  includes/config/tp.config.php, includes/libraries/csrfp/libs/csrfp.config.php, /includes/sk.php, /includes/teampass-seckey.txt from the former installation directory to the new one
4. Rename new directory as per vhost config
5. Attempt upgrade

### Expected behaviour
Successful upgrade

### Actual behaviour
Upgrade keeps asking for a SALT key, but since we started fresh with 2.1.27.0, sk.php is empty and we only have contents on teampass-seckey.txt (i also have to point that the documentation does not instruct to copy this last file).

### Server configuration
**Operating system**: 
GNU/Linux CentOS 7

**Web server:**
Apache 2.4.6

**Database:**
Percona MySQL 5.7.17-11

**PHP version:**
7.1.3

**Teampass version:** 
2.1.27.0

**Updated from an older Teampass or fresh install:**
Fresh install

### Client configuration
**Browser:**
Chrome 58

**Operating system:**
Windows 10

**Comment by nilsteampassnet** [comment]
Can you please download from [Master ](https://github.com/nilsteampassnet/TeamPass/archive/master.zip)and not 2.1.27.4?
Restore dump from previous version.
Then overwrite the files and restart upgrade

**Comment by nilsteampassnet** [comment]
You should have a selection to do

![image](https://user-images.githubusercontent.com/1197546/27654934-e12782a0-5c43-11e7-9153-1ff1f9ea2b41.png)

**Comment by mrsaraiva** [comment]
Hello Nils,

I'll try with Master and report back. Thanks!

**Comment by mrsaraiva** [comment]
Hi,

I got that combobox and the upgrade went on sucessfully, but i had two minor issues that had to be manually corrected:

First, it modified settings.php and removed the port for MySQL. The other problem was that it copied my teampass-seckey.txt to a different name (teampass-seckey.txt.2017_06_28.1498684464) and created a new one with a different hash teampass-seckey.txt, so passwords were not visible. I removed the new one and renamed the current one back and passwords worked.
